pkg/cri/containerd/client.go

```diff
@@ -0,0 +1,11 @@
+package containerd
+
+import (
+	"context"
+
+	"github.com/containerd/containerd"
+)
+
+type Client interface {
```

Comment:
I understand that you have to use different name than `ContainerRuntime` here since they are in the same package, but this is really the `ContainerRuntime` also the `containerd` is the implentation, 

Let's have under `pkg/cri`:

```golang
type ContainerRuntime interface {
  LoadContainer(ctx context.Context, id string) (containerd.Container, error)
}
```

Then we can bave the implementation with the same name under `pkg/cri/containerd`

Reply:
With the `LoadContainer` call ?

Reply:
These are 2 different interfaces: one is a container runtime, which can be different runtimes: crio / containerd /...
The other, is a containerd client, since I am mocking it in this unit test.

If this is confusing, I can drop the test, which removes the need to mock the containerd client.

Reply:
My fault the containerd concept is called Client https://github.com/containerd/containerd/blob/main/client.go#L85